frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -1,31 +1,78 @@
 # Getting Started
 
+- [Cypress Cucumber Hand book](https://docs.google.com/document/d/1hL_k5r6CVxbY5va6RPPCJfndQjLwSdt6SUKj-kWLqig/edit#)
+
+## Setup
+
+### Include Extensions
+
+"alexkrechik.cucumberautocomplete",
+"dylanmoerland.cypress-cucumber-steps"
+"streetsidesoftware.code-spell-checker"
+
+Add below content in settings.json under .vscode folder
+
+{
+"cucumberautocomplete.steps": [
+"./frontend/packages/dev-console/integration-tests/support/step-definitions/*/*.ts",
+"./frontend/packages/dev-console/integration-tests/support/step-definitions/*.ts"
+],
+"cucumberautocomplete.strictGherkinCompletion": true,
+"editor.quickSuggestions": true,
+"editor.defaultFormatter": "esbenp.prettier-vscode",
+"[javascript]": {
+"editor.defaultFormatter": "esbenp.prettier-vscode"
+}
+}
```

Comment:
Wrap the json with some code comment. The code comment could also contain a language hint: ```json .... ```

Otherwise it looks like this:

![image](https://user-images.githubusercontent.com/139310/106261131-8208f500-6221-11eb-8d15-b845d1a45661.png)

```suggestion
```json
{
  "cucumberautocomplete.steps": [
    "./frontend/packages/dev-console/integration-tests/support/step-definitions/*/*.ts",
    "./frontend/packages/dev-console/integration-tests/support/step-definitions/*.ts"
  ],
  "cucumberautocomplete.strictGherkinCompletion": true,
  "editor.quickSuggestions": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  }
}
```
```